src/main/java/uk/gov/hmcts/reform/em/stitching/service/impl/DocumentTaskServiceImpl.java

```diff
@@ -88,13 +75,17 @@ public DocumentTaskDTO save(DocumentTaskDTO documentTaskDTO) {
     }
 
     /**
-     * Delete the documentTask by id.
+     * Use the task processor to process the task
      *
-     * @param id the id of the entity
+     * @param documentTaskDTO task to process
+     * @return updated dto
      */
     @Override
-    public void delete(Long id) {
-        log.debug("Request to delete DocumentTask : {}", id);
-        documentTaskRepository.deleteById(id);
+    public DocumentTaskDTO process(DocumentTaskDTO documentTaskDTO) {
+        DocumentTask documentTask = documentTaskMapper.toEntity(documentTaskDTO);
+
+        itemProcessor.process(documentTask);
```

Comment:
is this the bit that makes it synchronous now?

Reply:
yep